src/react/OverflowItem.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { useMergedRefs } from "@fluentui/react-northstar";
+import { useOverflowItem } from './useOverflowItem';
+
+export type OverflowItemChildProps = React.HTMLAttributes<HTMLElement> & {
+  ref?: React.Ref<never>;
+};
+
+export type OverflowItemProps = {
+  id: string | number;
+  groupId?: string | number;
+  priority?: number;
+  children: React.ReactElement;
```

Comment:
I followed the other components in Fluent v9 that enforce a single child here vs. the intersection of ReactNode and render functions and null.